debian/dist/Debian/rules

```diff
@@ -216,7 +220,9 @@ binary-indep: build install
 	dh_installexamples -i
 	dh_installmenu -i
 	dh_installlogrotate -i
+	dh_systemd_enable -i support-files/mariadb.service support-files/mariadb.socket
 	dh_installinit -i
+	dh_systemd_start -i --restart-after-upgrade mariadb.socket
```

Comment:
does this require systemd? dh_systemd_enable does not, according to its man page.